.gitignore

```diff
@@ -18,12 +19,15 @@ lib/
 lib64/
 parts/
 sdist/
+tests/
```

Comment:
Was this a leftover change or intentional? I believe we should not be ignoring the `tests` folder, right?

Reply:
Thanks, good catch! I was removing some artifacts and then did the parent-level directory. I forgot these have actual tests in them. Just pushed a commit to be more specific with these exclusions.